lesson_2.ipynb

```diff
@@ -0,0 +1,2129 @@
+{
```

Comment:
<p>Great idea plotting just a section of the data, focuses on just one part of the trend which is great and wont overwehlm people too much. You also picked a great part of the series to showcase these as the third point from the left is "out of band"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='113'/>